vm_api/src/util/mod.rs

```diff
@@ -57,6 +57,33 @@ pub fn apply_code<S: Serialize>(
     assert_eq!(code, res.code, "expected code {}, got {} ({})", code, res.code, res.message);
     res.ret.map_or(RawBytes::default(), |b| RawBytes::new(b.data))
 }
+
+pub fn apply_ok_implicit<S: Serialize>(
+    v: &dyn VM,
+    from: &Address,
+    to: &Address,
+    value: &TokenAmount,
+    method: MethodNum,
+    params: Option<S>,
+) -> RawBytes {
+    apply_code_implicit(v, from, to, value, method, params, ExitCode::OK)
+}
```

Comment:
There's no need for this indirection since I can't think of any case where we expect an implicit message to not return OK. So just inline the body here.